### ISSUE TITLE
Any chance to update this to new SvelteKit?

### ISSUE THREAD

**Issue by WebShapedBiz** [rodneylab/sveltekit-session-cookies#3]
Thank you for this example but it's hard to follow and fix things with the new version of SvelteKit. Any chance for an update?

Cheers.

**Comment by rodneylab** [comment]
Done! Sorry for the delay.  See [new example using the current SvelteKit routing API](https://github.com/rodneylab/svelte-login-form).

**Comment by rodneylab** [comment]
Closing as update is now available.
